README.rst

```diff
@@ -183,6 +187,16 @@ is 3.4.5.
 
 .. _Semantic Versioning: http://semver.org/
 
+Version 0.7.0.20.0
+~~~~~~~~~~~~~~~~~~
+
+To be released.
+
+- Changed ``multi`` option to allow numeric values as well.
+  e.g. ``iterfzf(list, multi=15)`` allows the selection of at most 15 items.
+  [`#11`__ by Abraham Hinteregger]
+
+__ https://github.com/dahlia/iterfzf/pull/11
```

Comment:
Could you move the changelog to the below section, <q>Version 0.6.0.20.0</q>, which has not been released yet?